nixos/maintainers/scripts/ec2/amazon-base-config.nix

```diff
@@ -1,5 +1,5 @@
 { modulesPath, ...}:
 {
-  imports = [ "${modulesPath}/virtualisation/amazon-config.nix" ];
+  imports = [ ../../../../nixos/modules/virtualisation/amazon-image.nix ];
```

Comment:
Without this, we get infinite recursion. I don't want `<nixpkgs/` here because that might not match my current evaluation.